C0T0/Classes/ApiDataSource/URLRequestBuilder.swift

```diff
@@ -18,12 +18,21 @@ class StandardURLRequestBuilder: URLRequestBuilder {
     }
 
     func build(from request: ApiRequest) -> URLRequest? {
-        guard let  url = URL(string: host + request.endpoint) else { return nil }
+        var urlParametersString = ""
+        if let urlParameters = request.urlParameters {
+            for (key, value) in urlParameters {
+                urlParametersString += "&\(key)=\(value)"
+            }
+        }
+
+        guard let  url = URL(string: host + request.endpoint + urlParametersString) else { return nil }
```

Comment:
I think there is a small error. Current implementation adds `&` before every parameter, so before first parameter too. imo the sign which separates endpoint and parameters is `?`, not `&`.